packages/cqrs/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.0.0
+
+- **Breaking:** Add `propertyName` field in `ValidationError`.
+
```

Comment:
I would include the past versions (5.0.0 prereleases) here. They do exist on pub.dev so imo they should be here